parliament_challenge.py

```diff
@@ -0,0 +1,111 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def filter_speeches_dict(speeches):
+    """
+    Filter speeches data for relevant items.
+    """
+
+    speeches_list = []
+    for s in speeches:
+        speechesDict = {
+            "anforande_id": s.get("anforande_id"),
+            "dok_datum": s.get("dok_datum"),
+            "parti": s.get("parti"), 
+            "avsnittsrubrik": s.get("avsnittsrubrik"),
+            "links": [{ 
+                "rel": "speech",
+                "href": s.get("protokoll_url_www")
+                    }],
+            "intressent_id": s.get("intressent_id")
+            }
+        speeches_list.append(speechesDict)
+    return speeches_list
+
+
+def filter_member_dict(member):
+    """
+    Filter member data for relevant items.
+    """
+
+    membersDict = {
+        "tilltalsnamn": member.get("tilltalsnamn"),
+        "valkrets": member.get("valkrets"),
+        "bild_url_192": member.get("bild_url_192"),
+        "intressent_id": member.get("intressent_id")
+    }
+    return membersDict
+
+
+def get_member_data(intressent_id):
+    """
+    Get memeber data filter for relevant items.
+    """
+
+    domain = 'http://data.riksdagen.se'
+    members = 'personlista'
+    format_type = 'json'
+    response = requests.get('{}/{}/?iid={}&utformat={}'.format(domain, members, intressent_id, format_type))
+    if response.status_code == 200:
```

Comment:
What about the else part? Never forget the else part.

Reply:
return []